ui/lib/apps/ContinuousProfiling/pages/ConProfSettingForm.tsx

```diff
@@ -21,12 +22,24 @@ import { useClientRequest } from '@lib/utils/useClientRequest'
 import { ErrorBar, InstanceSelect } from '@lib/components'
 import { useIsWriteable } from '@lib/utils/store'
 
+const ONE_DAY_SECONDS = 24 * 60 * 60
 const RETENTION_SECONDS = [
-  3 * 24 * 60 * 60,
-  5 * 24 * 60 * 60,
-  10 * 24 * 60 * 60,
+  3 * ONE_DAY_SECONDS,
+  5 * ONE_DAY_SECONDS,
+  10 * ONE_DAY_SECONDS,
 ]
 
+function translateSecToDay(seconds: number, t: TFunction) {
+  // in our case, the seconds value must be the multiple of one day seconds
+  if (seconds % ONE_DAY_SECONDS !== 0) {
+    console.warn(`${seconds} is not the mulitple of one day seconds`)
+  }
+  const day = seconds / ONE_DAY_SECONDS
+  return t('continuous_profiling.settings.profile_retention_duration_option', {
+    d: day,
+  })
+}
+
```

Comment:
How did we format them in the time selector? In time selector seems that we don't need to explicitly write xxx days as a translation key.

Reply:
Yep, because we happen to show 7/14/28 days in the time selector, if we choose to show 10 days, it will display `1 week, 3 day` as here.

![image](https://user-images.githubusercontent.com/1284531/140303304-b08e61da-8ca6-4ae8-9b19-3792180e7ae9.png)

```
const RECENT_SECONDS = [
  // ...
  2 * 24 * 60 * 60,
  3 * 24 * 60 * 60,

  7 * 24 * 60 * 60,
  14 * 24 * 60 * 60,
  28 * 24 * 60 * 60,
]
```